src/commands/config.ts

```diff
@@ -0,0 +1,136 @@
+import { SlashCommandIntegerOption, SlashCommandStringOption } from '@discordjs/builders';
+import { codeBlock, isNullish } from '@sapphire/utilities';
+import { envParseArray } from '@skyra/env-utilities';
+import { Command, RegisterCommand, RegisterSubCommand } from '@skyra/http-framework';
+import { blue, bold, red, yellow } from '@skyra/logger';
+import { MessageFlags, PermissionFlagsBits } from 'discord-api-types/v10';
+
+@RegisterCommand((builder) =>
+	builder
+		.setName('config')
+		.setDescription("Manage a guild's features")
+		.setDMPermission(false)
+		.setDefaultMemberPermissions(PermissionFlagsBits.Administrator)
+)
+export class UserCommand extends Command {
+	@RegisterSubCommand((builder) => builder.setName('get').setDescription("Gets a guild's features").addStringOption(getGuildOption))
+	public async get(interaction: Command.ChatInputInteraction, options: Options) {
+		if (!UserCommand.ClientOwners.includes(interaction.user.id)) {
+			const content = 'You cannot use this command.';
+			return interaction.reply({ content, flags: MessageFlags.Ephemeral });
```

Comment:
```suggestion
			return interaction.reply({ content: 'You cannot use this command.', flags: MessageFlags.Ephemeral });
```

This is such a Kyra thing lmfao

Reply:
Actually, this is intentional. It's written line that to improve readability and visibility of the used functions.

Reply:
Well it doesn't achieve that

Reply:
This is still open